packages/splitter/splitter.stories.tsx

```diff
@@ -63,22 +66,25 @@ export const Container = ({
     return (
       <SplitterContainer
         defaultValueNow={defaultValueNow}
-        primaryPaneId={primaryPaneId}
         ariaLabel={ariaLabel}
         type={type}
         min={min}
         max={max}
         orientation={orientation}
         keyboardStep={keyboardStep}
+        position={position}
         rtl={rtl}
       >
-        {({ getSeparatorProps }: IUseSplitterReturnValue) => {
+        {({ getSeparatorProps, getPrimaryPaneProps, valueNow }: IUseSplitterReturnValue) => {
           const separatorProps = getSeparatorProps({
             style: {
               ...separatorStyle,
               cursor: orientation === SplitterOrientation.HORIZONTAL ? 'row-resize' : 'col-resize'
             }
           });
+          const primaryPaneProps = getPrimaryPaneProps({
+            style: primaryPaneStyle
```

Comment:
Is there a specific reason style was added here instead of below (in `style` prop)? Or vice versa? Wondering if they need to be split.

Reply:
Definitely looks like some duplication. Will simplify this. 👍

Reply:
Fixed in https://github.com/zendeskgarden/react-containers/pull/405/commits/04b21508933cf7ae55b3b61e795cb2c365789196